wrapper/src/test/java/integration2/container/MakeSureFirstInstanceWriterExtension.java

```diff
@@ -0,0 +1,45 @@
+package integration2.container;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import integration.util.AuroraTestUtility;
+import integration2.DatabaseEngineDeployment;
+import integration2.TestEnvironmentFeatures;
+
+public class MakeSureFirstInstanceWriterExtension implements BeforeAllCallback {
+
+  private final static Logger LOGGER = Logger.getLogger(MakeSureFirstInstanceWriterExtension.class.getName());
+
+  @Override
+  public void beforeAll(ExtensionContext context) throws Exception {
```

Comment:
should this be a @BeforeAll, or should we make the tests call the node rearrangement if they need the writer in position 0?